.circleci/config.yml

```diff
@@ -1,5 +1,36 @@
 version: 2
 jobs:
+  unit-php54-mage18:
+    docker:
+      - image: italoc/php54
```

Comment:
is there an official image by circle ci? those load faster and we know what we run

Reply:
Unfortunately not - I'm trying a few images but they all have some issues. I might need to toy around with one of the ones that is closest.

Reply:
I see. Ok. Ignore this